### PR TITLE
Fix running subset of existing ensemble

### PR DESCRIPTION
**Pull request by oyvindeide**
**Issue**
Resolves #my_issue

**Approach**
_Short description of the approach_

(Screenshot of new behavior in GUI if applicable)

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: src/ert/storage/local_ensemble.py [hunk 1]
```diff
@@ -199,28 +203,30 @@ def _responses_exist_for_realization(
             for response in self.experiment.response_configuration
         )
 
-    def is_initalized(self) -> bool:
+    def is_initalized(self) -> List[int]:
         """
         Check that the ensemble has all parameters present in at least one realization
```
Comment (frode-aarstad) [inline src/ert/storage/local_ensemble.py:208]: Update or remove comment?

File: src/ert/storage/local_ensemble.py [hunk 2]
```diff
@@ -199,28 +203,30 @@ def _responses_exist_for_realization(
...
                 (self._realization_dir(i) / f"{parameter}.nc").exists()
                 for parameter in self.experiment.parameter_configuration
             )
-            for i in range(self.ensemble_size)
         )
 
-    def has_data(self) -> bool:
+    def has_data(self) -> List[int]:
         """
         Check that the ensemble has all responses present in at least one realization
```
Comment (frode-aarstad) [inline src/ert/storage/local_ensemble.py:221]: Same as above

**Review by frode-aarstad — COMMENTED** [review]
Only some minor comments. Guess we can add test(s) for this later

**Review by frode-aarstad — APPROVED** [review]

File: src/ert/gui/ertwidgets/ensembleselector.py [hunk 3]
```diff
@@ -89,6 +90,12 @@ def _ensemble_list(self) -> Iterable[Ensemble]:
             )
         else:
             ensemble_list = self.notifier.storage.ensembles
+        ensemble_list = list(ensemble_list)
```
Comment (sondreso) [inline src/ert/gui/ertwidgets/ensembleselector.py:93]: Interesting line to read, one would think it was already a list :sweat_smile:

Comment (oyvindeide) [reply src/ert/gui/ertwidgets/ensembleselector.py:93]: One might think so... Could keep it a generator though

File: src/ert/gui/ertwidgets/ensembleselector.py [hunk 4]
```diff
@@ -21,6 +21,7 @@ def __init__(
         update_ert: bool = True,
         show_only_initialized: bool = False,
         show_only_undefined: bool = False,
+        show_only_no_children: bool = False,
```
Comment (dafeda) [inline src/ert/gui/ertwidgets/ensembleselector.py:24]: Should we add a comment describing what this does?
I giess we can infer from usage but not trivial I think.

**Pull request merged**